packages/plugin-loyalties-ui/src/loyalties/components/LoyaltySection.tsx

```
@@ -64,35 +64,42 @@ class LoyaltySection extends React.Component<IProps, State> {
-            {' '}
-            <Icon icon="star" />
-            {`Vouchers`} ({vouchers.length})
-          </li>
-        </Link>
+        {(ownerVouchers || []).map((ownerVoucher) => {
+          const { campaign, count } = ownerVoucher || {};
+
+          return (
+            <Link
+              to={`/vouchers?ownerId=${ownerId}&ownerType=${ownerType}&campaignId=${campaign._id}`}
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)